orchard-provider-aws/src/main/scala/com/salesforce/mce/orchard/io/aws/resource/Ec2Resource.scala

```diff
@@ -7,7 +7,7 @@
 
 package com.salesforce.mce.orchard.io.aws.resource
 
-import scala.jdk.CollectionConverters.CollectionHasAsScala
+import scala.jdk.CollectionConverters.{CollectionHasAsScala, SeqHasAsJava}
```

Comment:
can just do `scala.jdk.CollectionConverters._` for this one

Reply:
SeqHasAsJava is not needed.  It was used during refactoring before this version.  I have taken it out in new commit.